src/index.js

```diff
@@ -33,3 +64,26 @@ if ( document.readyState !== 'loading' ) {
   document.addEventListener( 'DOMContentLoaded', onReady );
 }
 
+/* 
+REFLECTION
+
+Challenges: 
+	* Knowing what is already in the DOM and what is being added. in renderTheUI(), my instinct 
```

Comment:
I think the framework would fit in when you need a direct and efficient relationship between data and the interface. Simply put, if there will be dynamic data, I want to instantly jump into React. Is that a crutch? Maybe?? But I don't see it being such a bad one if the tradeoff of relatively little bloat for ease of use is what i think it is. True?